models/request.js

```diff
@@ -0,0 +1,74 @@
+var User = require('./user');
+var Service = require('./service');
+module.exports = function(sequelize, DataTypes) {
+    var Request = sequelize.define("Request", {
```

Comment:
I'm realizing it will make more sense if we pluralize the model names since they're synonymous with table names.

So let's change both instances of `Request` to `Requests` here.

It makes more sense with the Sequelize method naming conventions like `Requests.findOne()` and `Requests.findAll()`